blog/models.py

```diff
@@ -185,9 +187,20 @@ def save(self, *args: Any, **kwargs: Any) -> None:  # pylint: disable=signature-
 
     def clean(self) -> None:
         super().clean()
+        self._clean_recommended_articles()
         self._validate_title_length()
         self._validate_recommended_articles_uniqueness()
 
+    def _clean_recommended_articles(self) -> Any:
```

Comment:
This method is returning `None` why you add `Any` here?

Reply:
Ah shots! I've forgotten about it! It's a leftover from different approach 😖

Reply:
Fixed it 👌